cumulusci/cli/tests/test_run_task.py

```diff
@@ -0,0 +1,343 @@
+"""Tests for the RunTaskCommand class"""
+
+from cumulusci.cli.cci import RunTaskCommand
+import click
+import pytest
+from unittest.mock import Mock, patch
+
+from cumulusci.cli import cci
+from cumulusci.core.config import BaseProjectConfig
+from cumulusci.core.exceptions import CumulusCIUsageError, CumulusCIException
+from cumulusci.cli.tests.utils import run_click_command, DummyTask
+
+
+@pytest.fixture
+def runtime():
+    runtime = Mock()
+    runtime.get_org.return_value = (None, None)
+    runtime.project_config = BaseProjectConfig(
+        None,
+        config={
+            "tasks": {
+                "dummy-task": {"class_path": "cumulusci.cli.tests.utils.DummyTask"},
+                "lots-o-options-task": {
+                    "class_path": "cumulusci.cli.tests.utils.MultipleOptionsTask"
+                },
+                "dummy-derived-task": {
+                    "class_path": "cumulusci.cli.tests.test_run_task.DummyDerivedTask"
+                },
+            }
+        },
+    )
+    return runtime
+
+
+def test_task_run(runtime):
+    DummyTask._run_task = Mock()
+    multi_cmd = cci.RunTaskCommand()
+    with patch("cumulusci.cli.cci.RUNTIME", runtime):
```

Comment:
You could make the `runtime` fixture take care of doing this patching.

Reply:
if I set the end of the `runtime` fixture to:
```
with patch('cumulusci.cli.cci.RUNTIME', runtime):
    return runtime
```
I'm not seeing the patch take place when I execute the test. Anything wrong with this approach?

Reply:
@Br4nd0R You need to do `yield runtime` instead of `return runtime` so that the patch context manager exits after pytest is done using the fixture, rather than immediately before returning runtime.